web/packages/teleport/src/Roles/RoleEditor/StandardEditor/standardmodel.ts

```diff
@@ -1193,6 +1375,9 @@ function optionsModelToRoleOptions(model: OptionsModel): RoleOptions {
     case 'deprecated-on':
       options.port_forwarding = true;
       break;
+
+    default:
+      mode satisfies '';
```

Comment:
TIL, this feels a bit better than having a separate case branch for `''` and then making sure that the default branch satisfies `never`.